### DISCUSSION TITLE
Ground Truth for Medications

### DISCUSSION THREAD

**Discussion by aegis301** [MIT-LCP/mimic-code#1464] [MIMIC-IV]
Dear MIMIC Community,

I am wondering if there is a "ground truth" in terms of medications that were actually given to a patient? So far I was using emar for this but now I read that this is based on a newer system and some older entries might not have been included there? mimic_iv.inputevents seems to be lacking information as well since I am missing a lot of medications here which I then wind up to find in mimic_iv.emar. Are there best practices in place?

Kind regards

**Comment by alistairewj** [comment]
* emar is an excellent ground truth for administration, *but*, it doesn't exist for half of the patients in the database. Checking if an `hadm_id` has at least one row in *emar* is a good way to check if they have data there - and if they do, you're good to go.
* inputevents is manually documented by nurses at the bedside in the ICU. It only pertains to a controlled list of medications (around ~250 or so if I recall). So, if the patient is in the ICU, and the medication is a part of the controlled list, then it is an excellent source of truth. However, you've clearly found a case where the medication isn't included (IV steroids are a good example of a drug that isn't included IIRC)
* There isn't another source of administration in the database - but you can always use *prescriptions*, which gives intention to treat.
